### PR TITLE
Feature pre release bugfixes

### PR DESCRIPTION
**Pull request by villejuutila**

### CODE REVIEW

File: src/utils/html-resource-utils.ts [hunk 1]
```diff
@@ -108,13 +109,31 @@ namespace HtmlResourceUtils {
     const elementClone = treeObject.element.cloneNode(false) as HTMLElement;
 
     for (const childNode of treeObject.element.childNodes) {
-      if (childNode.nodeType === Node.TEXT_NODE) {
+      // TODO: Move these to constants
```
Comment (anttileppa) [inline src/utils/html-resource-utils.ts:112]: Maybe this TODO should be done?

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: src/components/layout/v2/button-component-properties.tsx [hunk 2]
```diff
@@ -61,29 +62,81 @@ const ButtonComponentProperties = ({
...
+
+  /**
+   * Event handler for font size change events
+   *
+   * @param name name
+   * @param value value
+   */
+  const onFontSizeChange = ({ target: { name, value } }: ChangeEvent<HTMLInputElement>) => {
+    if (!value) {
+      component.element.style.fontSize = "0px";
```
[the remark this answers is not in the document]
Comment (Jdallos) [reply src/components/layout/v2/button-component-properties.tsx:105]: Font size has a default size applied initially (if no fontSize has already been set). That is why the font size gets set to 0 in the case that the user deletes the value in the form input.

So either the font size is set to 0 in this case or in order to remove the property we would need to remove the default font size functionality. Which way do you think is best?

File: src/components/layout/v2/text-component-properties.tsx [hunk 3]
```diff
@@ -102,6 +103,49 @@ const TextComponentProperties = ({
...
 
+  /**
+   * Event handler for font size change events
+   *
+   * @param name name
+   * @param value value
+   */
+  const onFontSizeChange = ({ target: { name, value } }: ChangeEvent<HTMLInputElement>) => {
+    if (!value) {
+      component.element.style.fontSize = "0px";
```
[the remark this answers is not in the document]
Comment (Jdallos) [reply src/components/layout/v2/text-component-properties.tsx:114]: Same as above.

File: src/components/layout/v2/button-component-properties.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (anttileppa) [reply src/components/layout/v2/button-component-properties.tsx:105]: Ok, I'll resolve this comment

**Review by anttileppa — APPROVED** [review]

**Pull request merged**